tip-wlan/charts/wlan-prov-service/templates/deployment.yaml

```diff
@@ -53,7 +53,7 @@ spec:
                 name: {{ include "common.fullname" . }}-creds
                 key: schema-repo-password
           image: postgres:latest
```

Comment:
While we are at it, it may be worthwhile to remove the hardcoded image and version and define these in values.yaml.